app/src/sharedTest/java/org/oppia/android/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -104,6 +107,8 @@ import javax.inject.Singleton
 )
 class AdministratorControlsActivityTest {
 
+  private val internalProfileId = 0
```

Comment:
Let's use `internalProfileId` at all places where needed, Thanks for adding this variable

```kt
      createAdministratorControlsActivityIntent(
        profileId = 0
      )
```

Reply:
Done as suggested here:  https://github.com/oppia/oppia-android/pull/3152/commits/81190dcf51b553fa939f95dc7525f6f1b4dec464